libstuff/SHTTPSManager.cpp

```diff
@@ -54,6 +54,30 @@ int SStandaloneHTTPSManager::getHTTPResponseCode(const string& methodLine) {
     return 400;
 }
 
+string SStandaloneHTTPSManager::getHeaderValue(const SData& fullResponse, const string& headerKey) {
+
+    const string fullResponseString = fullResponse.serialize();
```

Comment:
An SData is a deserialized HTTP message. It has fields `methodLine`, `nameValueMap`, and `content`. Of these,`nameValueMap` is the headers. If you want a header out of an `SData`, you can get it with:

SData request = // something.

// Get the Content-Type header.
string contentType = request["Content-Type"];